build.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+source ./.env
+set -e +u
+set -o pipefail
+
+IMG="lidofinance/oracle:latest"
+export DOCKER_CONFIG=$HOME/.lidofinance
+
+PUSH=false
```

Comment:
We use 1 or 0 for binary options throughout the project.

if `PUSH` either doesn't exist or `PUSH=0` we don't try to push.
if `PUSH` exists but value is neither 0 nor 1 we should give the message of unrecognized value.
if `PUSH=1` we try to push.